tests/k8s_tool.py

```diff
@@ -0,0 +1,50 @@
+import os
+import time
+import sys
+from kubernetes import config
+from kubernetes import client
+
+
+def load_config():
+    kube_config_var = os.getenv('KUBECONFIG', '~/.kube/config')
+    kube_config_path = os.path.expanduser(kube_config_var)
+    if os.path.isfile(kube_config_path):
+        config.load_kube_config(kube_config_path)
+    else:
+        raise Exception("KUBECONFIG file not found: {0}".format(kube_config_path))
+
+# Usage : python3.7 k8s_tool.py restarts namespace_name 10
+def find_restarted_pods(namespace):
+    namespaced_pods = client.CoreV1Api().list_namespaced_pod(namespace=namespace, include_uninitialized=True).items
+    pods = {}
+    for pod in namespaced_pods:
+        if pod is not None and pod.status is not None and pod.status.container_statuses is not None:
```

Comment:
How can pod.status be None if pod is not none

Reply:
Because some members can still be not initialized even if `pod` is initialized.

Reply:
ok